www/about.xslt

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="ISO-8859-15"?>
```

Comment:
Somewhat disappointing that there's so much boilerplate for this, I guess there's no other option though?

Reply:
abstracted this